addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1055,6 +1057,11 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         if (!value) {
             return;
         }
+        this.$target[0].dataset.requirementComparator = "";
+        this.$target[0].dataset.customError = "";
+        this.$target[0].dataset.errorMessage = "";
+        this.$target[0].dataset.requirementCondition = "";
+        this.$target[0].dataset.requirementBetween = "";
```

Comment:
Maybe it's me but I prefer this:
```suggestion
        Object.assign(this.$target[0].dataset, {
            requirementComparator: "",
            customError: "",
            errorMessage: "",
            requirementCondition: "",
            requirementBetween: "",
        });
```
What do you think? And probably the same approach later in the code...